force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,9 +195,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
-        Integer milliseconds;
-        return null; // Replace null with the variable you used to store the result
+        final Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
```

Comment:
You did great using the "final" keyword to make this variable a constant. That first code scanner complaint is just because constants are typically defined at the class level. The second alert is because you aren't actually using it. Set the variable equal to the number of milliseconds per minute and then use it in your calculation below.